### ISSUE TITLE
Use RJSF schema from Sistent in Meshery UI

### ISSUE THREAD

**Issue by Yashsharma1911** [meshery/meshery#10445]
### Mockups

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by Yashsharma1911** [comment]
I'll be adding more detailed info about this issue in an hour

**Comment by NAMANIND** [comment]
please add more details ~
 I will love to contribute

**Comment by Yashsharma1911** [comment]
Hey @NAMANIND I missed this message, are you still up to pick this issue?

**Comment by Yashsharma1911** [comment]
Hey thanks for raising your hand @11happy can you confirm you have build and able to run Meshery locally?

**Comment by 11happy** [comment]
Yes I can confirm have the build locally , I recently also published some designs for meshery catalog using `mesheryctl`. should I attach the image of local build?
